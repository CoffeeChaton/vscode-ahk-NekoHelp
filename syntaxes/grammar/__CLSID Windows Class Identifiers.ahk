~F1::
    Run ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
    Run ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Opens the Recycle Bin.
    Run ::{450d8fba-ad25-11d0-98a8-0800361b1103}\My Folder  ; Opens a folder that exists inside "My Documents".
    Run, %A_MyDocuments%\My Folder  ; Same as the above on most systems.

    FileSelectFile, OutputVar,, ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Select a file in the Recycle Bin.
    FileSelectFolder, OutputVar, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Select a folder within My Computer.
Return


~F2::
Run ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
Run ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Opens the Recycle Bin.
Run ::{450d8fba-ad25-11d0-98a8-0800361b1103}\My Folder  ; Opens a folder that exists inside "My Documents".
Run, %A_MyDocuments%\My Folder  ; Same as the above on most systems.

FileSelectFile, OutputVar,, ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Select a file in the Recycle Bin.
FileSelectFolder, OutputVar, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Select a folder within My Computer.
Return

~F12:: Run, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
