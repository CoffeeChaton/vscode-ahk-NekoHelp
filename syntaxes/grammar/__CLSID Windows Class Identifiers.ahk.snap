>~F1::
#^ source.ahk meta.hotkey.line.ahk storage.modifier.hotkey.ahk
# ^^ source.ahk meta.hotkey.line.ahk meta.2.hotkey.line.ahk markup.bold.hotkey.ahk
#   ^^ source.ahk meta.hotkey.line.ahk
>    Run ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
#^^^^ source.ahk
#    ^^^ source.ahk meta.command.line.ahk support.function.ahk
#       ^ source.ahk meta.command.line.ahk
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                ^ source.ahk meta.command.line.ahk
#                                                 ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>    Run ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Opens the Recycle Bin.
#^^^^ source.ahk
#    ^^^ source.ahk meta.command.line.ahk support.function.ahk
#       ^ source.ahk meta.command.line.ahk
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                ^ source.ahk meta.command.line.ahk
#                                                 ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                  ^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>    Run ::{450d8fba-ad25-11d0-98a8-0800361b1103}\My Folder  ; Opens a folder that exists inside "My Documents".
#^^^^ source.ahk
#    ^^^ source.ahk meta.command.line.ahk support.function.ahk
#       ^ source.ahk meta.command.line.ahk
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                ^^^^^^^^^^^ source.ahk meta.command.line.ahk
#                                                           ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>    Run, %A_MyDocuments%\My Folder  ; Same as the above on most systems.
#^^^^ source.ahk
#    ^^^ source.ahk meta.command.line.ahk support.function.ahk
#       ^ source.ahk meta.command.line.ahk punctuation.ahk
#        ^ source.ahk meta.command.line.ahk
#         ^ source.ahk meta.command.line.ahk punctuation.definition.variable.percent.ahk
#          ^^^^^^^^^^^^^ source.ahk meta.command.line.ahk variable.other.constant.builtin.ahk
#                       ^ source.ahk meta.command.line.ahk punctuation.definition.variable.percent.ahk
#                        ^^^^^^^^^^^ source.ahk meta.command.line.ahk
#                                   ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>    FileSelectFile, OutputVar,, ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Select a file in the Recycle Bin.
#^^^^ source.ahk
#    ^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#                  ^ source.ahk meta.command.line.ahk punctuation.ahk
#                   ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                             ^ source.ahk meta.command.line.ahk punctuation.ahk
#                              ^ source.ahk meta.command.line.ahk punctuation.ahk
#                               ^ source.ahk meta.command.line.ahk
#                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                                        ^ source.ahk meta.command.line.ahk
#                                                                         ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>    FileSelectFolder, OutputVar, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Select a folder within My Computer.
#^^^^ source.ahk
#    ^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#                    ^ source.ahk meta.command.line.ahk punctuation.ahk
#                     ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                               ^ source.ahk meta.command.line.ahk punctuation.ahk
#                                ^ source.ahk meta.command.line.ahk
#                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                                         ^ source.ahk meta.command.line.ahk
#                                                                          ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Return
#^^^^^^ source.ahk keyword.control.flow.ahk
>
>
>~F2::
#^ source.ahk meta.hotkey.line.ahk storage.modifier.hotkey.ahk
# ^^ source.ahk meta.hotkey.line.ahk meta.2.hotkey.line.ahk markup.bold.hotkey.ahk
#   ^^ source.ahk meta.hotkey.line.ahk
>Run ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
#^^^ source.ahk meta.command.line.ahk support.function.ahk
#   ^ source.ahk meta.command.line.ahk
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                            ^ source.ahk meta.command.line.ahk
#                                             ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Run ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Opens the Recycle Bin.
#^^^ source.ahk meta.command.line.ahk support.function.ahk
#   ^ source.ahk meta.command.line.ahk
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                            ^ source.ahk meta.command.line.ahk
#                                             ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                              ^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Run ::{450d8fba-ad25-11d0-98a8-0800361b1103}\My Folder  ; Opens a folder that exists inside "My Documents".
#^^^ source.ahk meta.command.line.ahk support.function.ahk
#   ^ source.ahk meta.command.line.ahk
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                            ^^^^^^^^^^^ source.ahk meta.command.line.ahk
#                                                       ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Run, %A_MyDocuments%\My Folder  ; Same as the above on most systems.
#^^^ source.ahk meta.command.line.ahk support.function.ahk
#   ^ source.ahk meta.command.line.ahk punctuation.ahk
#    ^ source.ahk meta.command.line.ahk
#     ^ source.ahk meta.command.line.ahk punctuation.definition.variable.percent.ahk
#      ^^^^^^^^^^^^^ source.ahk meta.command.line.ahk variable.other.constant.builtin.ahk
#                   ^ source.ahk meta.command.line.ahk punctuation.definition.variable.percent.ahk
#                    ^^^^^^^^^^^ source.ahk meta.command.line.ahk
#                               ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>FileSelectFile, OutputVar,, ::{645ff040-5081-101b-9f08-00aa002f954e}  ; Select a file in the Recycle Bin.
#^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#              ^ source.ahk meta.command.line.ahk punctuation.ahk
#               ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                         ^ source.ahk meta.command.line.ahk punctuation.ahk
#                          ^ source.ahk meta.command.line.ahk punctuation.ahk
#                           ^ source.ahk meta.command.line.ahk
#                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                                    ^ source.ahk meta.command.line.ahk
#                                                                     ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>FileSelectFolder, OutputVar, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Select a folder within My Computer.
#^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk support.function.ahk
#                ^ source.ahk meta.command.line.ahk punctuation.ahk
#                 ^^^^^^^^^^ source.ahk meta.command.line.ahk
#                           ^ source.ahk meta.command.line.ahk punctuation.ahk
#                            ^ source.ahk meta.command.line.ahk
#                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                                     ^ source.ahk meta.command.line.ahk
#                                                                      ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>Return
#^^^^^^ source.ahk keyword.control.flow.ahk
>
>~F12:: Run, ::{20d04fe0-3aea-1069-a2d8-08002b30309d}  ; Opens the "My Computer" folder.
#^ source.ahk meta.hotkey.line.ahk storage.modifier.hotkey.ahk
# ^^^ source.ahk meta.hotkey.line.ahk meta.2.hotkey.line.ahk markup.bold.hotkey.ahk
#    ^^ source.ahk meta.hotkey.line.ahk
#      ^ source.ahk
#       ^^^ source.ahk meta.command.line.ahk support.function.ahk
#          ^ source.ahk meta.command.line.ahk punctuation.ahk
#           ^ source.ahk meta.command.line.ahk
#            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk string.windows_class_identifiers.ahk
#                                                    ^ source.ahk meta.command.line.ahk
#                                                     ^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk
#                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.command.line.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>